ouroboros-network.cabal

```diff
@@ -93,10 +93,10 @@ test-suite tests
   main-is:             Main.hs
   other-modules:       Test.Chain
                        Test.ChainProducerState
-                       Test.InCtxt
                        Test.Node
                        Test.Sim
                        Test.GlobalState
+                       Test.ArbitrarySt
```

Comment:
What do people think about giving these test modules a more distinct name space? Like maybe all of these test modules could be called `Test.Ouroboros.X`  so when people see them, they will have a good clue about where they came from. This naming convention is not that important now, but when this package is imported into another (and say generators from this package are used elsewhere) naming can significantly help code readability.

Reply:
There is no namespace _at all_ for the library itself, even. Yeah, that'll eventually need to be fixed, agreed.